forage-android/src/main/java/com/joinforage/forage/android/collect/BTPinCollector.kt

```diff
@@ -32,6 +34,7 @@ internal class BTPinCollector(
             path = balancePath(paymentMethodRef)
         }
 
+        logger.i("Sending balance check to BasisTheory")
```

Comment:
include `merchantAccount`,  `paymentMethodRef` and `cardToken` in log?

Reply:
https://github.com/teamforage/forage-android-sdk/pull/42#discussion_r1274193522

should omit cardToken